docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2469,6 +2469,44 @@ definitions:
 
           If this is set it MUST be a valid required field as defined in the schema.
 
+      cleanup_policy:
+        type: string
+        x-extensible-enum:
+          - delete
+          - compact
+        default: 'delete'
+        description: |
+          Event type cleanup policy. There are two possible values:
+
+          - 'delete' cleanup policy will delete old events after retention time expires. Nakadi guarantees that each
+          event will be available for at least the retention time period. However Nakadi doesn't guarantee that event
+          will be deleted right after retention time expires.
+
+          - 'compact' cleanup policy will keep only the latest event for each event key. The compaction is performed per
```

Comment:
I know we strive to decouple Nakadi from Kafka, but I think it would be informative to link to the section in Kafka docs that talk about consumption. A user reading this thing about "head" and "recent" might not have the same context as we do and for that reason have many questions. I think we could point there and say that for now this feature is only supported when using Kafka as a backend.

Reply:
Yes, makes sense. Added the link https://github.com/zalando/nakadi/pull/905/commits/ff56c178009f86fa1265685bb9a35c5e39e41897